exporter/kafkaexporter/config.go

```diff
@@ -64,6 +67,14 @@ type Config struct {
 	Authentication kafka.Authentication `mapstructure:"auth"`
 }
 
+type HeaderInjection struct {
+	// Whether to inject headers or not, default is false.
+	InjectHeaders bool `mapstructure:"inject_headers"`
```

Comment:
Is this boolean needed? You might not need that, just check any headers are present.